repo/remote.go

```diff
@@ -389,20 +385,28 @@ func (r *RemoteCache) Mod(importPath string) (modPath, name string, err error) {
 	}
 
 	// Ask "go list".
-	v, err := r.mod.ensure(importPath, func() (interface{}, error) {
-		modPath, err := r.ModInfo(importPath)
+	mvalue := func() (interface{}, error) {
+		modPath, err = r.ModInfo(importPath)
 		if err != nil {
 			return nil, err
 		}
 		return modValue{
 			path: modPath,
 			name: label.ImportPathToBazelRepoName(modPath),
 		}, nil
-	})
+	}
+
+	v, err := r.mod.ensure(importPath, mvalue)
 	if err != nil {
 		return "", "", err
 	}
```

Comment:
Is this part needed? Seems to be a no-ops style change?

Reply:
nice catch, fixed